lib/init/javascript.js

```diff
@@ -33,14 +31,14 @@ exportables.loadNpm = () => {
 };
 
 // Resolve an npm cofig list, or nothing (existance is not needed)
-exportables.getNpmConfig = (npm) => {
+exportables.resolveNpmConfig = (npm) => {
```

Comment:
In general, I think things that are "resolved" by a promise should be named accordingly.